src/test/java/org/checkstyle/autofix/recipe/UpperEllRecipeTest.java

```diff
@@ -3,38 +3,53 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.openrewrite.java.Assertions.java;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 import org.junit.jupiter.api.Test;
 import org.openrewrite.test.RecipeSpec;
 import org.openrewrite.test.RewriteTest;
 
 public class UpperEllRecipeTest implements RewriteTest {
 
+    private static final String BASE_TEST_RESOURCES_PATH = "src/test/resources/org"
+            + "/checkstyle/autofix/recipe/";
+
     @Override
     public void defaults(RecipeSpec spec) {
         spec.recipe(new UpperEllRecipe());
     }
 
     @Test
-    void fixesLowercase() {
-        rewriteRun(
-
-            java(
-                "class Test {\n"
-                        + "    int value1 = 123l;\n"
-                        + "    long value2 = 0x123l;\n"
-                        + "    long value3 = 0123l;\n"
-                        + "    long value4 = 0b101l;\n"
-                        + "    String value5 = null;\n"
-                        + "}\n",
-                "class Test {\n"
-                        + "    int value1 = 123L;\n"
-                        + "    long value2 = 0x123L;\n"
-                        + "    long value3 = 0123L;\n"
-                        + "    long value4 = 0b101L;\n"
-                        + "    String value5 = null;\n"
-                        + "}\n"
-            )
-        );
+    void test1() throws IOException {
+        final String beforeCode = Files.readString(Paths.get(BASE_TEST_RESOURCES_PATH
+                        + "upperell/before/InputUpperEllTest.java"))
+                .replaceFirst("^\\s*package.*?;\\s*\\n?", "");
+
+        final String afterCode = Files.readString(Paths.get(BASE_TEST_RESOURCES_PATH
+                        + "upperell/after/InputUpperEllTest.java"))
+                .replaceFirst("^\\s*package.*?;\\s*\\n?", "");
+
+        rewriteRun(java(beforeCode, afterCode));
+
+        assertTrue(true, "Test completed successfully");
```

Comment:
Remove this assertion and wrap `rewriteRun` into the assertion that checks no exception is thrown, please check junit5 docs for this